docs/EthStats/Lite-Network-Health-Monitor.md

```diff
@@ -0,0 +1,119 @@
+description: Lite Network Health Monitor
+<!--- END of page meta data -->
+
+# Lite Network Health Monitor
+
+Use the EthStats Lite Network Health Monitor to monitor the health of private networks by displaying real time and 
+historical statistics about the network nodes.
+
+The lite version supports in-memory persistence or using Redis to persist a fixed number of blocks (by default,
+3000). 
+
+View the Network Health Monitor for the [Ethereum MainNet](https://net.ethstats.io)
+
+!!! note 
+     The EthStats Lite Block Explorer is an [Alethio product](https://aleth.io/).
+
```

Comment:
Also perhaps note the list of ports that will be used in this tutorial as we use static local port 80 and 3000 and there's many chances of conflicts with existing services.

Reply:
Added